js/demo.js

```diff
@@ -0,0 +1,17 @@
+import $ from 'jquery';
```

Comment:
Main reason for loading templates via XHR is the demo pages in `ig-article` aren't actually used for anything other than depicting how the page should render (AFAICT, anyway). For things to be useful in something like `starter-kit`, the header and footer need to be in partial templates so they can be included via `gulp-html-tag-include`.